chart/values.yaml

```diff
@@ -28,6 +28,10 @@ namespaceOverride: ""
 ##
 fullnameOverride: ""
 
+ssl-exporter:
```

Comment:
It would be great, if you could add some info about this new options into the `README.md`.